src/sagemaker/utils.py

```diff
@@ -1449,3 +1449,12 @@ def get_instance_type_family(instance_type: str) -> str:
         if match is not None:
             instance_type_family = match[1]
     return instance_type_family
+
+
+def is_likely_a_pandas_df(data: Any) -> bool:
+    """Return True if data is likely a pandas dataframe."""
+    return (
+        data.__class__.__name__ == "DataFrame"
+        and hasattr(data, "columns")
+        and hasattr(data, "iloc")
+    )
```

Comment:
I think we must test this utility function.

We could use [`pytest.importorskip`](https://docs.pytest.org/en/latest/how-to/skipping.html#skipping-on-a-missing-import-dependency) in the test:

```python
pd = pytest.importorskip("pandas")
```